src/components/modules/dragNDrop.ts

```diff
@@ -119,4 +218,47 @@ export default class DragNDrop extends Module {
   private processDragOver(dragEvent: DragEvent): void {
     dragEvent.preventDefault();
   }
+
+  /**
+   * Create drag image for drag-n-drop.
+   *
+   * @param blocks {Block[]} - blocks to create drag image for.
+   * @returns {HTMLElement} - drag image.
+   */
+  private createDragImage(blocks: Block[]): HTMLElement {
+    const { BlockManager } = this.Editor;
+
+    /**
+     * If we are dragging only one block, return its content.
+     */
+    if (blocks.length == 1) {
+      const block = BlockManager.currentBlock;
+
+      return block.holder.querySelector(`.${Block.CSS.content}`);
+    }
+
+    /**
+     * If we are dragging multiple blocks, create a drag image with all blocks content.
+     */
+    const dragImage: HTMLElement = document.createElement('div');
```

Comment:
We have `DOM.make` for creating elements

Reply:
done

Reply:
done